api/test.go

```diff
@@ -0,0 +1,42 @@
+package api
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/vocdoni/arbo"
+	"github.com/vocdoni/arbo/memdb"
+	"github.com/vocdoni/vocdoni-z-sandbox/state"
+)
+
+var tree *arbo.Tree
+var testAvailable bool
```

Comment:
There is no need for this bool, instead you can check if 'testProcessTree' is nil.